package.json

```diff
@@ -29,6 +31,12 @@
     "rewire-test-helpers": "^0.2.0",
     "rimraf": "^2.4.3",
     "sinon": "^1.17.2",
+    "standard": "^10.0.2",
     "webpack": "^1.12.2"
+  },
+  "standard": {
+    "env": [
```

Comment:
Note that this applies the mocha env to all files being linted, including source (non-test) files.  If we'd prefer, we can switch to [using comments at the head of files](https://standardjs.com/#what-about-mocha-jasmine-qunit-etc)